cmsplugin_cascade/bootstrap3/image.py

```diff
@@ -48,7 +48,7 @@ class BootstrapImagePlugin(LinkPluginBase):
     name = _("Image")
     model_mixins = (ImagePropertyMixin, LinkElementMixin,)
     module = 'Bootstrap'
-    parent_classes = ['BootstrapColumnPlugin']
+    parent_classes = ['BootstrapColumnPlugin','BootstrapPanelPlugin']
```

Comment:
there is no need to add a ``BootstrapPanelPlugin``, since it a considered "transparent". This works really well in version 0.12, so please test your environment against the latest development version.